simplq/src/components/common/HeaderSection/headerSection.module.scss

```diff
@@ -2,23 +2,13 @@
 
 .header-bar {
   @include background-color();
-  min-height: 8.5rem;
-  padding: 0.5rem 0 0 4.5rem;
+  padding: 6rem 2rem 2rem 2rem;
   display: flex;
   flex-wrap: wrap;
-  .simplq-logo {
-    padding-top: 1.1rem;
-    z-index: 3;
-  }
+  align-items: flex-end;
+  justify-content: space-evenly;
   .queue {
-    margin: auto;
-    position: absolute;
-    top: 1rem;
-    bottom: 0;
-    left: 0;
-    right: 0;
     align-items: center;
-    z-index: 1;
     .header-title {
```

Comment:
How did you manage without z-index? Did the align-items: flex-end fix the problem we were having before?

Reply:
I don't know, I just removed the z-index and everything was working fine. Can you see if you can spot any issues?

I don't exactly remember the reason why we put the z-indices. Maybe the underlying issue got resolved due to some other change.

Reply:
Oh okay